### PR TITLE
Fixed missing exposition of organization_update from cmds

### PR DESCRIPTION
**Pull request by c4ffein**

### CODE REVIEW

File: parsec/core/backend_connection/administration.py [hunk 1]
```diff
@@ -5,32 +5,19 @@
...
-
-        return wrapper
-
-    ping = _expose_cmds("ping")
-
-    organization_create = _expose_cmds("organization_create")
-    organization_status = _expose_cmds("organization_status")
-    organization_stats = _expose_cmds("organization_stats")
+    for cmd_name in ADMINISTRATION_CMDS:
+        vars()[cmd_name] = expose_cmds(cmd_name)
```
Comment (touilleMan) [inline parsec/core/backend_connection/administration.py]: SWAG is over 9000 on this :trollface:

File: tests/backend/test_anonymous.py [hunk 2]
```diff
@@ -40,24 +42,7 @@ def not_decorated():
 
 @pytest.mark.trio
 async def test_anonymous_has_limited_access(anonymous_backend_sock):
```
Comment (touilleMan) [inline tests/backend/test_anonymous.py]: would be usefull to also have a `test_authenticated_has_limited_access`

(we could typically keep all the 3 tests inside the same `tests/backend/test_access.py` file given how related they are)

File: newsfragments/1032.bugfix.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+Added missing organization_update to admin cmds, refactored methods definition at that occasion
```
Comment (touilleMan) [inline newsfragments/1032.bugfix.rst]:
```
Add missing `organization_update` to backend administration api
```

No need to talk about the refactoring given it's internal stuff

**Comment by touilleMan** [comment]
image

File: tests/backend/test_access.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+async def test_anonymous_has_limited_access(anonymous_backend_sock):
+    await check_forbidden_cmds(
+        anonymous_backend_sock, (ADMINISTRATION_CMDS | AUTHENTICATED_CMDS) - ANONYMOUS_CMDS
+    )
+
+
+@pytest.mark.trio
+async def test_authenticated_has_limited_access(alice_backend_sock):
+    await check_forbidden_cmds(
+        alice_backend_sock, (ADMINISTRATION_CMDS | AUTHENTICATED_CMDS) - AUTHENTICATED_CMDS
```
Comment (touilleMan) [inline tests/backend/test_access.py]: copy/paste error here ;-)

Comment (c4ffein) [reply tests/backend/test_access.py]: Fixed!

**Comment by c4ffein** [comment]
Closes #1034

**Pull request merged**